src/scripts/Api.js

```diff
@@ -0,0 +1,25 @@
+class Api {
```

Comment:
* Please, delete one redundant `Api.js` from the code https://snipboard.io/jk7b5e.jpg